### ISSUE TITLE
Individual plots - QA

### ISSUE THREAD

**Issue by Mutaz94** [UUPharmacometrics/PsNR#6]
Hi @rikardn 

While generating `qa` report. It looks like there is something going on with `individual_plots_extra` function that terminates rendering the document

```
Quitting from lines 497-502 (PsN_qa_plots.Rmd) 
Error in individual_plots_extra(file_name = original_max0_model, ID_nr = ii_list$infl_id[1:10],  : 
  could not find function "individual_plots_extra"
Calls: <Anonymous> ... handle -> withCallingHandlers -> withVisible -> eval -> eval
```

I used the latest version of both PsN and PsNR. Thanks!
Mutaz

**Comment by rikardn** [comment]
Thanks for reporting this. The latest released version of PsN does not work together with the latest version of PsNR. If installing PsN using the installer you should get the version of PsNR that works together with PsN. The PsN installer comes bundled with the correct version of PsNR.

**Comment by Mutaz94** [comment]
Thanks for getting into this. I will try to match the release since when I tried it at the beginning, the error was `PsNR is not installed`

**Comment by rikardn** [comment]
This issue might be related to https://github.com/UUPharmacometrics/PsN/issues/203

**Comment by Mutaz94** [comment]
Yes. I initially got the same error but once I downloaded the newer version moved to the other error. I will try to work around it and post it here! Thanks!!
